doc/source/rllib/index.rst

```diff
@@ -20,20 +20,22 @@ making needs.
 
 You **don't need** to be an **RL expert** to use RLlib, nor do you need to learn Ray or any
 other of its libraries! If you either have your problem coded (in python) as an
-`RL environment <https://medium.com/distributed-computing-with-ray/anatomy-of-a-custom-environment-for-rllib-327157f269e5>`_
+`RL environment <rllib-env.html#configuring-environments>`__
```

Comment:
We shouldn't internally link to HTML; could we instead use a cross-reference? [docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html)

Reply:
All the other urls on the github rllib README are full urls.  I'll just replace those new urls with full urls for now, instead of refactoring whole page.